client/src/components/Header.css

```diff
@@ -11,19 +14,44 @@ header a:visited {
 
 .header-logo_container {
 	display: flex;
-	align-items: flex-end;
 	flex-shrink: 0;
-	gap: 2.125em;
+	gap: 2em;
 	margin-left: 5.125em;
 }
 
-.header-text {
+.header-logo-decodenoir {
 	font-family: "Oswald", sans-serif;
 	font-weight: bold;
 	font-size: 2em;
 }
 
-.header-logo_cyf {
+.header-logo-cyf {
 	width: 12.5em;
 	height: 3.75em;
 }
+
+@media screen and (max-width: 768px) {
+	/* header {
+        align-items: center; 
+        display: flex;
+        flex-direction: column; 
+        height: auto; 
+    }
+    .header-logo_container {
+        align-items: center;
+        display: flex;
+        flex-direction: column; 
+        margin: 2%;
+    } */
+
+	header {
+		justify-content: center;
+		align-items: center;
+	}
+
+	.header-logo_container {
+		align-items: center;
+		display: flex;
+		margin: 2%;
```

Comment:
Since our sections below have a padding of 16px on the side, it would be better to use a margin of 16px to ensure they match.
16px / 2% = 800px, so this gap would be slightly smaller on all mobile screen widths